src/SIL.XForge.Scripture/locales.json

```diff
@@ -82,5 +82,11 @@
     "englishName": "Western Kayah Li",
     "tags": ["kyu", "kyu-Kali"],
     "production": true
+  },
+  {
+    "localName": "नेपाली",
+    "englishName": "Nepali",
+    "tags": ["npi", "ne-NP"],
+    "production": true
```

Comment:
Why is this immediately enabled before anything is localized? Is there a rationale for that, or was this just copying the other examples? Turning it off in production allows us to test and make sure fonts, and the initial localizations, etc., are working, before turning it on for users. It also reduces the probably that we get told it's "not working" because none of the strings are localized yet.